### PR TITLE
Rebranding to GoPlay

### PR DESCRIPTION
**Pull request by michaelarnauts**
* Fixes for rebranding to GoPlay
* Disabled categories for now, since they don't seem to be used anymore

Download zips:
[plugin.video.viervijfzes-0.3.1-goplay-8de921a.zip](https://github.com/add-ons/plugin.video.viervijfzes/files/5889776/plugin.video.viervijfzes-0.3.1-goplay-8de921a.zip)
[plugin.video.viervijfzes-0.3.1+matrix.1-goplay-8de921a.zip](https://github.com/add-ons/plugin.video.viervijfzes/files/5889777/plugin.video.viervijfzes-0.3.1%2Bmatrix.1-goplay-8de921a.zip)

### CODE REVIEW

File: resources/lib/viervijfzes/__init__.py [hunk 1]
```diff
@@ -37,14 +37,30 @@
...
+        name='Play 6',
+        epg_id='zes',
+        url='https://www.goplay.be',
+        logo='play6.png',
+        background='play6-background.png',
         iptv_preset=6,
-        iptv_id='zes.be',
+        iptv_id='play6.be',
+        youtube=[],
+    )),
```
Comment (timrijckaert) [inline resources/lib/viervijfzes/__init__.py:49]: Play 7? 😲

**Comment by frankdpGH** [comment]
Thanks for this remediation, working on my side.

**Comment by wputman29** [comment]
@michaelarnauts , last time I checked yesterday, the tv guide didn't sync (yet) and it returned me an error when I refreshed the channels and guides in IPTV Manager, can you confirm that too?

**Comment by michaelarnauts** [comment]
@wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.

https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)

**Comment by wputman29** [comment]
> @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> 
> https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)

Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?

**Comment by michaelarnauts** [comment]
> > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> 
> @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> 
> Reset of my kodi did the job. Thanks anyway ;)

I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

**Comment by wputman29** [comment]
> > > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> > 
> > 
> > @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> > Reset of my kodi did the job. Thanks anyway ;)
> 
> I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

When clearing Kodi completely from scratch, it works fine with the new logo, but from the moment you import 'old' userdata, even when the app is not installed yet, the old logo comes back once you install it. I even tried removing all the userdata from the viervijfzes addon, it still happens. A clean wipe of all userdata did it, now it works fine with the new logo. Let's hope they keep it for a while ;)

**Comment by MPParsley** [comment]
Thank you very much for this update!

Not sure if this is related but I'm still experiencing some playback issues. I'm not seeing any indication of previously played items and somehow I can play all episodes from Masterchef S06 up until episode 12. As of 13 and up the video can't be played.

Tried both zip files mentioned in this PR and uninstalled/reinstalled the app.

**Comment by dagwieers** [comment]
> I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?

In the past we found that only when the add-on was updated in the Kodi repository, the add-on would reliably be changed in Kodi everywhere.

File: resources/lib/modules/channels.py [hunk 2]
```diff
@@ -7,10 +7,9 @@
 
 from resources.lib import kodiutils
 from resources.lib.kodiutils import TitleItem
-from resources.lib.modules.menu import Menu
 from resources.lib.viervijfzes import CHANNELS, STREAM_DICT
 from resources.lib.viervijfzes.auth import AuthApi
-from resources.lib.viervijfzes.content import CACHE_AUTO, CACHE_ONLY, ContentApi
+from resources.lib.viervijfzes.content import ContentApi
```
Comment (MPParsley) [inline resources/lib/modules/channels.py:12]: Was this done intentionally?

**Review by MPParsley — APPROVED** [review]

**Comment by vermope** [comment]
@wputman29 When updating the metadata from the settings menu all logos are renewed and visible.

**Comment by michaelarnauts** [comment]
> > > > @wputman29 Can you try with this zip, it's the latest code of this branch. I got IPTV Manager to work, but I don't know if I've updated the zips with that fix.
> > > > https://github.com/add-ons/plugin.video.viervijfzes/archive/goplay.zip (Leia only)
> > > 
> > > 
> > > @michaelarnauts Works fine, thank you :) . Just one remark, even though the icon and fanart are renewed in the zip, it still shows me the old 4 5 6 logos one on the video add-on logo, even after reinstalling. Is it somehow still referring to the old one before because this is just a fix, or am I missing something?
> > > Reset of my kodi did the job. Thanks anyway ;)
> > 
> > 
> > I have also noticed this. It seems Kodi caches the add-on image. Not sure when Kodi updated this. Maybe after a restart?
> 
> When clearing Kodi completely from scratch, it works fine with the new logo, but from the moment you import userdata, even when the app is not installed yet, the old logo comes back once you install it. I even tried removing all the userdata from the viervijfzes addon, it still happens. Is it possible IPTV manager caches that logo too, it's the only addon that I can think of that would use the data of it in order to send it to the PVR for the tv guide? Only a clean wipe of all userdata did it in the end, now it works fine with the new logo. Let's hope they keep it for a while ;)

I've also had this on my development machine, and I fixed it there by removing the Textures.db file in the Kodi userdata/database folder while Kodi wasn't running.

**Comment by wputman29** [comment]
Thanks for checking 😊

**Pull request merged**